File: src/main/java/com/dsm/quiz/model/user/controller/AuthController.java
package com.dsm.quiz.model.user.controller;

import com.dsm.quiz.model.user.dto.UserCreateRequestDto;
import com.dsm.quiz.model.user.service.AuthService;
import lombok.RequiredArgsConstructor;
import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.ResponseStatus;
import org.springframework.web.bind.annotation.RestController;

@RequiredArgsConstructor
@RestController
public class AuthController {

    private final AuthService authService;

    @PostMapping("/auth")
    @ResponseStatus(HttpStatus.CREATED)
    public void join(UserCreateRequestDto user){
        authService.join(user);
    }

    @PostMapping("/user")
    public void login(){

    }

}

File: src/main/kotlin/com/quiz/qq/domain/user/controller/AuthController.kt
package com.quiz.qq.domain.user.controller

import com.quiz.qq.domain.user.dto.UserCreateRequestDto
import com.quiz.qq.domain.user.service.AuthService
import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.ResponseStatus
import org.springframework.web.bind.annotation.RestController

@RestController
class AuthController(private val authService: AuthService) {

    @ResponseStatus(HttpStatus.CREATED)
    @PostMapping("users")
    fun join(@RequestBody dto: UserCreateRequestDto){
        authService.join(dto);
    }

}